src/Microsoft.DncEng.Configuration.Extensions/Microsoft.DncEng.Configuration.Extensions.csproj

```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <IsPackable>true</IsPackable>
-    <Nullable>enable</Nullable>
```

Comment:
Why not keep this?

Reply:
Because there are a bunch of warnings I couldn't remove. You really can't add nullable to the _middle_ of a tech stack, because stuff below it can't be annotated so you end up in irreconcilable situations.  If we want to enable it, we shouldn't do it piecemeal in random projects, and rather have a real plan (which we can't afford)

Reply:
Isn't it possible to do a slow adoption with the `#nullable` directive? These classes are already annotated, and I'd hate to lose that knowledge.

Reply:
The problem is they are incorrectly annotated.  You can slowly do it, but you have to start at the rootmost assembly (so that changes in base assemblies don't produce irreconcilable errors in assemblies that depend on them).  In my opinion, nullable costs more than is save anyway.

It also needs to be a plan to do it everywhere. Having to check which "kind" of assembly you are in, and having to check if you are supposed to slap "?" and "!" all over the place as you navigate around the project costs, me at least, a ton of time and frustration for the limit benefit it provides.  That's fine as a temporary state if the end goal is to use it everywhere, but I don't think the value is there for us to do that, and the costs of having a split code base are higher than just not having it anywhere.

Reply:
This one actually looks like it was a base-most assembly... but it wasn't annotated correctly. So it wasn't a bad place to start... but we do need to commit to continuing (and actually do it correctly in the project we enable it for... not just enable it but leave all the violations).  Turning on nullable... but then disabling all the warnings it produces seems... counterproductive?

Reply:
That all makes sense.